web/src/utils/transformDataHelper.tsx

```diff
@@ -146,4 +150,14 @@ export function transformJsonToCsv(objArray: any) {
     str += line + '\r\n';
   }
   return str;
+};
+
+export function getProfileMinistry(ministrySet: any[], profileDetails: any): object {
+  let ministryDetails: any = {};
+  ministrySet.forEach((ministry: any) => {
```

Comment:
Alternatively, we could have something like`ministrySet.filter((ministry:any) => {ministry.id === profileDetails.busOrgId})`. We could also throw exception so the react component that calls it could utilize err msg.

Reply:
Good catch, I'll implement this as a bug fix after merge.